src/bindgen/ir/structure.rs

```diff
@@ -232,6 +232,47 @@ impl Source for Struct {
         if config.language == Language::Cxx {
             let mut wrote_start_newline = false;
 
+            if config.structure.derive_constructor(&self.annotations)
+                && !self.fields.is_empty()
+            {
+                if !wrote_start_newline {
+                    wrote_start_newline = true;
+                    out.new_line();
+                }
+
+                out.new_line();
+
+                let arg_renamer = RenameRule::GeckoCase;
```

Comment:
Could we use `config.function.rename_args` here? Just so it's customizable.

Reply:
Updated patch to pull out a lambda that uses the config.function.rename_args if it exists.